igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/stash/client/StashClient.groovy

```diff
@@ -31,5 +34,23 @@ interface StashClient {
         @Path('projectKey') String projectKey,
         @Path('repositorySlug') String repositorySlug,
         @QueryMap Map queryMap)
+
+  // TODO: pagination support
```

Comment:
I don't think this is a big deal for now as I don't think we'll have that many manifests.